bloom-housing/bloom
# Adds Jurisdiction to listings, users, translations (replaces countyCode)

# Pull Request Template

## Issue

Addresses # #1614


## Description

This PR covers adding jurisdiction relantionships to listings, users and translations, which replaces the use of countyCode. You'll note that countyCode is kept as a virtual field, so I wouldn't have to update every instance in the front-end apps. There is now a `JURISDICTION_NAME` environment variable for the public app. The value can be the same as COUNTY_CODE, which can be removed.

## Type of change


## How Can This Be Tested/Reviewed?

You should be able to create a new user from the public app, and they should be assigned to the jurisdiction set in the env var. On the partner's side, you should be able to create a new listing. If the user has access to more than one jurisdiction, they should see a dropdown to select the jurisdiction at the top of the form, otherwise it should set to the user's jurisdiction and not show the dropdown.

This does not address adding different roles for users under different jurisdictions. We can add that when we need it.


## Checklist:
